src/Shared/Recruit.Vacancies.Client/Domain/Entities/SourceType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
+{
+    public enum SourceType
+    {
+        Clone,
+        Extension,
+        New

```

Comment:
maybe `Original` even though ticket says `New`

Reply:
@sachpatel the property uses the word "User" because it reflects which user last updated the vacancy (ie. a person)
if there was a system update to the vacancy then this wouldn't be recorded using these field(s)